src/dbnode/storage/bootstrap/bootstrapper/commitlog/source_index_test.go

```diff
@@ -25,28 +25,42 @@ import (
 	"testing"
 	"time"
 
+	"github.com/m3db/m3/src/dbnode/namespace"
```

Comment:
Probably not in this PR for speed of merging, but should probably merge all of these with their respective `source_data_test` functions